foca/errors/errors.py

```diff
@@ -88,3 +90,14 @@ def __handle_internal_server_error(exception: Exception) -> Response:
         status=500,
         mimetype="application/problem+json"
     )
+
+
+def __handle_invalid_specs(exception: Exception) -> Response:
```

Comment:
Actually, `InvalidSpecification` should never be raised in an app context, but _only_ when starting or restarting the service. Therefore it shouldn't return a response nor be attached to a handler attached. Since the error is already defined by Connexion, there is not much to do other than raising `InvalidSpecification(msg)` in the code, with a fitting, descriptive error message.

In case I said sth else before, I'm sorry for that, I didn't think it through properly.